devbox.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packages": [
+    "nodejs@18.18.2",
+    "corepack@18.18.2",
+    "docker@latest"
```

Comment:
Would we want mountebank in here?

Reply:
Mountebank is not packaged and distributed by nix yet. So unable to include. Agree it would be good to have it here.